### PR TITLE
Add Readme and char/u8 const fn

### PR DESCRIPTION
**Pull request by homersimpsons**
This should fix `rust-lang/rust#82078` in #16

### CODE REVIEW

File: data/1.52/const_char_ascii_case.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
++++
+title = "`char::to_ascii_uppercase()`, `char::to_ascii_lowercase()` and `char::eq_ignore_ascii_case()` as `const fn`"
```
[the remark this answers is not in the document]
Comment (homersimpsons) [reply data/1.52/const_char_ascii_case.md]: Should be fixed in cbf33fd3045df778d5524ac0af96936f7be617eb

File: data/1.52/const_char_ascii_case.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
++++
```
[the remark this answers is not in the document]
Comment (homersimpsons) [reply data/1.52/const_char_ascii_case.md]: I gess the fourth one is the one from GitHub diff

File: data/1.52/const_char_eq_ignore_ascii_case.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
++++
+title = "`char::eq_ignore_ascii_case()` as `const fn`"
```
Comment (jplatte) [inline data/1.52/const_char_eq_ignore_ascii_case.md:2]:
```suggestion
title = "`char::eq_ignore_ascii_case` as `const fn`"
```

for consistency.

File: data/1.52/const_char_eq_ignore_ascii_case.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+impl_pr_id = 82078
+flag = "const_ascii_methods_on_intrinsics"
+items = [
+    "char::eq_ignore_ascii_case()",
+]
```
Comment (jplatte) [inline data/1.52/const_char_eq_ignore_ascii_case.md:7]: No need for `items` if the same info is already found in the title.

File: data/1.52/const_char_ascii_case.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jplatte) [reply data/1.52/const_char_ascii_case.md]: Oh, I guess the GitHub android app renders `+` and `-` while the website does not 🤦🏼

File: data/1.52/const_char_eq_ignore_ascii_case.md (hunk 3 above)
Comment (homersimpsons) [reply data/1.52/const_char_eq_ignore_ascii_case.md]: Should be fixed in cbf33fd3045df778d5524ac0af96936f7be617eb (I did update for the others too)

File: data/1.52/const_char_eq_ignore_ascii_case.md (hunk 4 above)
Comment (homersimpsons) [reply data/1.52/const_char_eq_ignore_ascii_case.md]: Okay, I did remove theses in cbf33fd3045df778d5524ac0af96936f7be617eb

**Comment by jplatte** [comment]
Squashed all changes except the readme into one commit: 99eeb90bd6b789a2246deb78c2d9ab6eac28c1f0
Also added a readme based on the one from this PR: 7f33ff0f117397c55187a0c003c44f57c431abe9

Thanks!

**Pull request closed without merging**